x-pack/test/security_solution_cypress/cypress/tasks/security_main.ts

```diff
@@ -9,7 +9,7 @@ import { CLOSE_TIMELINE_BUTTON, TIMELINE_BOTTOM_BAR_TOGGLE_BUTTON } from '../scr
 import { TIMELINE_EXIT_FULL_SCREEN_BUTTON, TIMELINE_FULL_SCREEN_BUTTON } from '../screens/timeline';
 
 export const openTimelineUsingToggle = () => {
-  cy.get(TIMELINE_BOTTOM_BAR_TOGGLE_BUTTON).click();
+  cy.get(TIMELINE_BOTTOM_BAR_TOGGLE_BUTTON).should('be.visible').click();
```

Comment:
this should not be necessary as the `click` action from Cypress should scroll to the element and ensure the element is not hidden before clicking (see [Cypress documentation](https://docs.cypress.io/guides/core-concepts/interacting-with-elements))